### DISCUSSION TITLE
Borderless Window at export and stretching does not work

### DISCUSSION THREAD

**Discussion by Nep-Toodles** [4ian/GDevelop#2353] [Enhancement ideas (and their technical discussions)]
BorderlessWindow is a window without border above and i really want it and also the fullscreen stretching doesnt work like stretch width just pushes the whole gameCanvas toi left in place of stretcvhing it and stretch height does the opposite 
- [ ] Window Borderless option in Exports
- [ ] Proper Stretching in Full screen 
- [x] When pressing CTRL + SHIFT + I it opens the inspector and it messes the editor up so remove it.
- [ ]  Textarea cause TextInput with fabricJS https://codepen.io/benheise/pen/dmOvPP?editors=1010
- [x] Get responce

**Comment by 4ian** [comment]
> Window Borderless option in Exports

Made a card for it on the roadmap, you can vote for it now:
https://trello.com/c/MPblTFMf/458-add-an-option-to-have-the-game-window-shown-without-borders-on-windows-macos-linux

> Textarea cause TextInput has to be programmed and left and right arrow ker dont work i should delete and go over there.

It's a bit hard to understand but I think I understood what you mean so here is also a new card on the roadmap to vote for:

https://trello.com/c/FSP58UTv/459-add-a-text-area-text-input-object-or-improve-the-text-entry-object-to-support-having-a-cursor-selection-multilines-etc

> When pressing CTRL + SHIFT + I it opens the inspector and it messes the editor up so remove it.

Does not seem a problem to me, it's only in GDevelop and a useful debugging tool.

> Proper Stretching in Full screen

You need to be more precise, include some screenshots and describe what you need.

**Comment by Nep-Toodles** [comment]
thanks

**Comment by Nep-Toodles** [comment]
Like I can mess up the whole editor by editing the HTML so Add a event that when a user press CTRL + SHIFT + I it will open help tab

**Comment by 4ian** [comment]
That's not a problem for the editor, as you are the developer and it's your problem if you mess up with the editor :) It's a feature, not a bug.
When you export your game, there is no such shortcuts, so your players are safe.

**Comment by Nep-Toodles** [comment]
Thanks but the feature is confusing but if u mind add a setting to turn it off in-case. also ok and why dogdevelop use canvas for scenes then using divs like other engines?

**Comment by 4ian** [comment]
The whole interface is made in React, rendering HTML element on screens. That's ideal for an interface. The games are rendered in WebGL, which is much more efficient for games.

**Comment by blurymind** [comment]
Its an interesting but niche idea. Godot has it but I haven't seen it used
a lot.
It can even render the window with transparent background so you can have
your sprite walk around the screen as if it escaped from the game.

Didn't electron have the option to spawn a borderless window? That won't
work for web export, but what would 🤔

**Comment by Nep-Toodles** [comment]
look at this video for borderless window : @blurymind https://www.youtube.com/watch?v=wiblQhPqXdY

**Comment by Nep-Toodles** [comment]
> The whole interface is made in React, rendering HTML element on screens. That's ideal for an interface. The games are rendered in WebGL, which is much more efficient for games.

ok thanks for the explaination it helps alot

**Comment by Nep-Toodles** [comment]
@4iacan you add this https://konvajs.org/docs/sandbox/Editable_Text.html or https://www.geeksforgeeks.org/how-to-create-a-simple-editable-canvas-textbox-using-fabric-js/   + Non draggable fabricJS https://stackoverflow.com/questions/14842877/make-fabric-js-object-draggable-but-not-selectable
@blurymind This is a fully working engine sorry for the ping if it distrubed you i edited the link cause it was not working
https://codepen.io/benheise/pen/dmOvPP?editors=1010 example of fabricJS
